src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1169,7 +1170,8 @@ export class NetworkAreaDiagramViewer {
             busNodeRadius,
             this.svgParameters.getNodeHollowWidth()
         );
-        const busElement: HTMLElement | null = <HTMLElement>node.children.item(busIndex);
+        const childIndex = this.svgParameters.getInsertNameDesc() ? busIndex + 1 : busIndex;
+        const busElement: HTMLElement | null = <HTMLElement>node.children.item(childIndex);
```

Comment:
We can instead do something like:
```typescript
node.querySelector('.nad-busnode:nth-of-type(' + busIndex + ')')`;
```

Reply:
Used query selector instead of children list, but not nth-of-type, it seems to work with elements of the same types, while here we have 'circle' and 'path' elements